src/app/personside/infotabs/meldinger/traadliste/TraadListe.tsx

```diff
@@ -119,16 +119,15 @@ function SlaaSammenTraaderKnapp({ traader }: { traader: Traad[] }) {
                 </KnappBase>
                 <StyledModalWrapper contentLabel={'Besvar flere'} onRequestClose={() => settApen(false)} isOpen={apen}>
                     <BesvarFlere
-                        traader={traaderMedTildelteOppgaver.sort(
+                        traader={traaderSomHarDelsvar.sort(
```

Comment:
hm, her skjønner jeg ikke helt hva som skjer... kan man bare slå sammen tråder som har delsvar..?

Reply:
Ifølge logikken i modiabrukerdialog så er det sånn ja.

Reply:
hm, det høres da veldig rart ut..? Man får vel heller ikke lov å slå sammen tråder som har mer enn et innslag, og siden et delsvar må følge et spørsmål vil den tråden ha minst to innslag og da får man i praksis ikke lov til å slå sammen noe som helst..?

Reply:
Hm, kom til å tenke på det: Dette ser veldig ut som funksjonaliteten for å slå sammen delsvar til et samlet svar når endelig svar foreligger. Sikker på at du ikke har sett på feil kontroller/kode..?

Reply:
Dette er knappen, hvis jeg ikke har missforstått (i Innboks.java):
`
slaaSammenTraaderToggleButton.add(visibleIf(new Model<Boolean>() {
            @Override
            public Boolean getObject() {
                return innboksVM.tildelteOppgaverUtenDelsvar.size() > 1;
            }
        }));
`
Og tildelteOppgaverUtenDelsvar bygges på denne måten:
`
        tildelteOppgaverUtenDelsvar.addAll(tildelteOppgaver.stream()
                .filter(oppgave -> !ofNullable(getTraader().get(oppgave.henvendelseId))
                        .map(TraadVM::harDelsvar)
                        .orElse(false))
                .collect(toList()));
`

Er det noe jeg missforstår @nutgaard ?

Reply:
Ah, det skal være uten delsvar...